scipy/optimize/_shgo.py

```diff
@@ -143,9 +143,23 @@ def shgo(func, bounds, args=(), constraints=None, n=None, iters=1,
 
         Objective function knowledge:
 
-        * symmetry : bool
-            Specify True if the objective function contains symmetric variables.
-            The search space (and therefore performance) is decreased by O(n!).
+        * symmetry : list or None
```

Comment:
If a `bool` was allowed for `symmetry` before, we need to continue to support that.